main/src/ca/uwaterloo/flix/language/fmt/DisplayType.scala

```diff
@@ -624,8 +624,16 @@ object DisplayType {
           }
 
         case TypeConstructor.SymmetricDiff =>
-          val args = t.typeArguments.map(visit)
-          SymmetricDiff(args)
+          t.typeArguments.map(visit).map(splitSymmetricDiff) match {
+            // Case 1: No args. ? ⊕ ?
+            case Nil => SymmetricDiff(Hole :: Hole :: Nil)
+            // Case 2: One arg. Take the left and put a hole at the end: tpe1 ⊕ ?
+            case arg :: Nil => SymmetricDiff(arg :+ Hole)
+            // Case 3: Multiple args. Concatenate them: tpe1 ⊕ tpe2
```

Comment:
I'd say "two args" here

i see now that the naming and comments you originally made was from the existing code, sorry about taxing you for it